descriptors/ModuleDescriptor-template.json

```diff
@@ -326,6 +326,11 @@
             "manualblocks.collection.get"
           ]
         },
+        {
+          "methods": ["POST"],
```

Comment:
can we specify multiple methods here? As there are multiple cases where we need to support PUT for example. Also, I think there is no need to distinguish circulation endpoint permission by method (i.e. `inn-reach.d2ir.circulation.item.post`). Lets just use `inn-reach.d2ir.circulation.all`. What do you think?

Reply:
Another option is to add a separate handler description for each circulation operation as the operations will need different set of `modulePermissions` most probably